extensions/amp-o2-player/OWNERS

```diff
@@ -4,7 +4,13 @@
 {
   rules: [
     {
-      owners: [{name: 'yevheniiminin'}, {name: 'ampproject/wg-components'}],
+      owners: [
+        {name: 'aharoyairncohen'},
+        {name: 'arielginsburg'},
+        {name: 'lironka'},
+        {name: 'adir-kandel-vzm'},
+        {name: 'ampproject/wg-components'},
```

Comment:
@lironka I asked for all the current player devs to be added. How come you've only added guys in Israel? You're missing myself, Suresh and people in Sunnyvale.